x-pack/plugins/apm/public/components/shared/backend_link.tsx

```diff
@@ -13,8 +13,19 @@ import { useApmRouter } from '../../hooks/use_apm_router';
 import { truncate } from '../../utils/style';
 import { ApmRoutes } from '../routing/apm_route_config';
 import { SpanIcon } from './span_icon';
+import { TruncateWithoutTooltip } from './truncate_with_tooltip';
 
-const StyledLink = euiStyled(EuiLink)`${truncate('100%')};`;
+const StyledLink = euiStyled(EuiLink)`min-width: 0`;
```

Comment:
What's the purpose of `min-width` if is set to `0`?

Reply:
Hey @kpatticha, the min width is required here because it is a flex item. Flex items takes content width by default, so giving min-width is necessary here.
Reference : https://css-tricks.com/flexbox-truncated-text/